### PR TITLE
Updates various messages for grenade-spiders giving more attention to themselves for others

### PR DESCRIPTION
**Pull request by Harrivchaw**
## About The Pull Request
all around standardization of carrion grenade-y spiders
warning message is now bold red and emits romnche sound

## Why It's Good For The Game
please i just want to see them again

## Changelog
:cl:
tweak: Grenade-like spiders (Toxic, Explosive, Flashbang) now emit more obvious tell-tale signs before activating
/:cl:

### CODE REVIEW

**Review by genessee596 — APPROVED** [review]
I actually asked him to do this change.

At the moment, the explosive pop-out means literally nothing, since half the time the person doesn't notice due to it being silent and giving no message.

Now, it gives the person a huge message (since they'd probably notice a massive, glowing spider crawling out they ass), and the spider praises arachnocommunism one last time before exploding. This, hopefully, gives the victim a good warning they're about to explode and gives them a second to close out of their menu and run. Or, you know, just type 'whats that noise' and proceed to get blown to smithereens. Their choice.

**Review by TorinoFermic — APPROVED** [review]

File: code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /obj/item/weapon/implant/carrion_spider/explosive
 	name = "explosive spider"
+	desc = "A spider that's ready to bites the dust."
```
Comment (TorinoFermic) [inline code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm:3]:
```suggestion
	desc = "A spider that's about to blow the dust."
```

**Comment by Iglpigl** [comment]
For future reference you should name your PRs appropriately in the future.

File: code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm [hunk 2]
```diff
@@ -1,5 +1,6 @@
 /obj/item/weapon/implant/carrion_spider/explosive
 	name = "explosive spider"
+	desc = "A spider that's about to blow the dust."
```
Comment (MadmanMartian) [inline code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm:3]: This description makes no sense

File: code/game/objects/items/weapons/implant/implants/carrion/flashbang_spider.dm [hunk 3]
```diff
@@ -1,13 +1,25 @@
 /obj/item/weapon/implant/carrion_spider/flashbang
 	name = "flashbang spider"
+	desc = "A spider filled with some sort of liquid, it feels warm."
```
Comment (MadmanMartian) [inline code/game/objects/items/weapons/implant/implants/carrion/flashbang_spider.dm:3]: This description isn't very descriptive.

File: code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm (hunk 2 above)
Comment (Harrivchaw) [reply code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm:3]: thought it was a clever reference to JoJo's Bizarre Adventure (Japanese: ジョジョの奇妙な冒険, Hepburn: JoJo no Kimyō na Bōken) a Japanese manga/anime series and Counter-Strike (CS) a series of multiplayer first-person shooter video games in which teams of terrorists battle to perpetrate an act of terror

but sure, what do you want it to be changed

Comment (MadmanMartian) [reply code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm:3]: >Jojo reference

It's bite the dust, not blow the dust.

Comment (Harrivchaw) [reply code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm:3]: i know, torino wanted it changed from bites the dust to blow the dust, which is in reference to de_dust2,  a video game map featured in the first-person shooter series Counter-Strike. Dust II is the successor to "Dust", another Counter-Strike map, and was developed by David Johnston prior to the official release of the original Counter-Strike game. It was designed with the aims of simplicity and balance, based on its symmetrical design and two points, for which the two teams must fight over for control.

i'll just get rid of the reference

Comment (TorinoFermic) [reply code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm:3]: Ah, I am sorry about it. I thought it was bad spelling.

Comment (Harrivchaw) [reply code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm]: it's alright, it's now replaced with the much more superior description

File: code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm [hunk 4]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/implant/carrion_spider/explosive
 	name = "explosive spider"
-	desc = "A spider that's about to blow the dust."
+	desc = "A spider that's bulgy, like it's about to burst. Better not put it in your pants."
```
Comment (MadmanMartian) [inline code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm:3]: What the f

Comment (Harrivchaw) [reply code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm:3]: oh wait i missed the perfect opportunity

File: code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm [hunk 5]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/implant/carrion_spider/explosive
 	name = "explosive spider"
-	desc = "A spider that's bulgy, like it's about to burst. Better not put it in your pants."
+	desc = "A spider that's bulgy, like it's about to blow on someone. Better not put it in your pants."
```
Comment (MadmanMartian) [inline code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm:3]: That's worse!

Comment (genessee596) [reply code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm:3]: Ahahahah what the fuck

Comment (TorinoFermic) [reply code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm:3]: Going for sexual innuendos ?

File: code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm [hunk 6]
```diff
@@ -1,5 +1,6 @@
 /obj/item/weapon/implant/carrion_spider/explosive
 	name = "explosive spider"
+	desc = "A spider that's bulgy, like it's about to blow on someone. Better not put it in your pants."
```
Comment (genessee596) [inline code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm:3]: please
```suggestion
	desc = "A large, glowing spider, about the size of your fist. It's undulating and emitting a soft ticking noise."
```

File: code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm (hunk 5 above)
Comment (Harrivchaw) [reply code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm:3]: speedmerge pls

**Review by genessee596 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm (hunk 6 above)
Comment (Harrivchaw) [reply code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm]: you break my heart

**Comment by Harrivchaw** [comment]
karen took my pr's name

**Comment by TorinoFermic** [comment]
> karen took my pr's name

Don't worry about it, this title is inside the commit message

**Pull request merged**